src/Symfony/Component/DomCrawler/Tests/Field/FormFieldTest.php

```diff
@@ -11,10 +11,26 @@
 
 namespace Symfony\Component\DomCrawler\Tests\Field;
 
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
 use Symfony\Component\DomCrawler\Field\InputFormField;
 
 class FormFieldTest extends FormFieldTestCase
 {
+    use ExpectDeprecationTrait;
+
+    /**
+     * @group legacy
+     */
+    public function testGetLabelIsDeprecated()
+    {
+        $this->expectDeprecation('Since symfony/dom-crawler 7.1: The "Symfony\Component\DomCrawler\Field\DomFormField::getLabel()" method is deprecated, use "Symfony\Component\DomCrawler\Field\DomFormField::getDomLabel()" instead.');
+
+        $node = $this->createNode('input', '', ['type' => 'text', 'name' => 'name', 'value' => 'value']);
+        $field = new InputFormField($node);
+
+        $field->getLabel();
```

Comment:
you should still assert the return value though, to ensure it still works properly.